TerminalDocs/group-policy.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Group Policy
+description: Learn how to set Group Policy for Windows Terminal.
+author: chrnguyen
+ms.author: chrnguyen
+ms.date: 10/24/2024
+ms.topic: how-to 
+---
+
+# Group Policies
+
+Since Windows Terminal Preview 1.22, Windows Terminal is released on GitHub with Administrative Templates that allow you to configure Windows Terminal using Group Policies.
+
+## How to install
+
+### Download
+
+You can find the latest administrative templates (ADMX and ADML files) in the assets section of our [newest Windows Terminal release on GitHub](https://github.com/microsoft/terminal/releases/latest). The file is named `GroupPolicyTemplates-<Version>.zip`
+
+### Add the administrative template to an individual computer
+
+1. Unzip the `GroupPolicyTemplates-<Version>.zip` file to your **Policy Definition** template folder. (Example: `C:\Windows\PolicyDefinitions`)
```

Comment:
Example makes no sense. There is only this one directory.

Reply:
You have a point about the usage of the word "example"
I will remove that word. I still think having the path is valuable for those who are new to setting up policies so I will change it to "located in" instead.

Reply:
I only meant that "Example" is useless.